build.sbt

```diff
@@ -2025,40 +1985,20 @@ lazy val `runtime-integration-tests` =
         (`profiling-utils` / javaModuleName).value,
         (`ydoc-server` / javaModuleName).value
       ),
-      Test / modulePath := {
-        val updateReport = (Test / update).value
-        val requiredModIds =
-          GraalVM.modules ++ GraalVM.langsPkgs ++ GraalVM.insightPkgs ++ logbackPkg ++ helidon ++ Seq(
-            "org.slf4j"           % "slf4j-api"                    % slf4jVersion,
-            "org.netbeans.api"    % "org-openide-util-lookup"      % netbeansApiVersion,
-            "org.netbeans.api"    % "org-netbeans-modules-sampler" % netbeansApiVersion,
-            "org.graalvm.sdk"     % "polyglot-tck"                 % graalMavenPackagesVersion,
-            "org.graalvm.truffle" % "truffle-tck"                  % graalMavenPackagesVersion,
-            "org.graalvm.truffle" % "truffle-tck-common"           % graalMavenPackagesVersion,
-            "org.graalvm.truffle" % "truffle-tck-tests"            % graalMavenPackagesVersion
-          )
-        val requiredMods = JPMSUtils.filterModulesFromUpdate(
-          updateReport,
-          requiredModIds,
-          streams.value.log,
-          shouldContainAll = true
-        )
-        val runtimeTestInstrumentsMod =
-          (`runtime-test-instruments` / Compile / exportedProducts).value.head.data
-        val runtimeMod =
-          (`runtime-fat-jar` / Compile / exportedProducts).value.head.data
-        val ydocMod =
-          (`ydoc-server` / Compile / exportedProducts).value.head.data
-        val syntaxMod =
-          (`syntax-rust-definition` / Compile / exportedProducts).value.head.data
-        val profilingMod =
-          (`profiling-utils` / Compile / exportedProducts).value.head.data
-        requiredMods ++ Seq(
-          runtimeTestInstrumentsMod,
-          runtimeMod,
-          ydocMod,
-          syntaxMod,
-          profilingMod
+      Test / moduleDependencies := {
+        GraalVM.modules ++ GraalVM.langsPkgs ++ GraalVM.insightPkgs ++ logbackPkg ++ helidon ++ Seq(
```

Comment:
This looks pretty understandable! It is a format I hope to be able to work with.

Reply:
I am glad. More info in the [scaladocs of JPMSPlugin](https://github.com/enso-org/enso/pull/10836/files#diff-298edb47ec128ff0addb0389107d18ee27e01c03e713e7f6b5fe15d1e0bbaef3R15-R30)